src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mens = [...people].filter(human => human.sex === 'm');
+
+  if (century) {
+    mens = mens.filter(men => Math.ceil(men.died / 100) === century);
+  }
```

Comment:
```suggestion
  let mens = [...people].filter(human => century
    ? human.sex === 'm' && Math.ceil(human.died / 100) === century
    : human.sex === 'm');
```
Также объедини в других функциях, чтобы избежать дополнительной фильтрации